src/bci_build/package/python.py

```diff
@@ -61,6 +72,10 @@ def _get_python_kwargs(
             Replacement(regex_in_build_description=pip3_replacement, package_name=pip3),
         ],
         "os_version": os_version,
+        "support_level": SupportLevel.L3,
+        "supported_until": _SLE_15_PYTHON_SUPPORT_ENDS[py3_ver]
+        if os_version in (OsVersion.SP4, OsVersion.SP5, OsVersion.SP6, OsVersion.SP7)
```

Comment:
This should be fine as we don't produce sp3 containers anyway?
```suggestion
        if os_version.is_sle15
```

Reply:
Yes  we have several places to clean up should the is_sle15 check merge